platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -330,6 +334,14 @@ public void addTrustedCertificate(
           e);
     } finally {
       IOUtils.closeQuietly(socket);
+      // close output stream
```

Comment:
You should use try with resources instead.
Replace the try first try with:
```
try(OutputStream fos = Files.newOutputStream(trustStoreFile)){
```

Reply:
It looks like a lot of work goes into getting that `trustStoreFile`. Should I try to move all of that logic before the try?

Or should I make a nested try with resources where the output stream is being initialized?

Or should I keep it as is?

Reply:
I see what you mean. You can use `IOUtils.closeQuietly(fos);` instead